### ISSUE TITLE
Critical bug: cache is not cleared onSave when backend language is not english

### ISSUE THREAD

**Issue by side-by-side** [rainlab/pages-plugin#156]
Hello.
When in user preferences backend language is set to other than english (russian in my case), on static page create/change cache is not cleared and such users must wait 10 minutes (accordingly to the setting `cms.parsedPageCacheTTL`).

Also, when new page is created and we trying to view it in frontend using its URL, this bug leads to “max execution time” php error in breadcrumbs component here: https://github.com/rainlab/pages-plugin/blob/master/components/StaticBreadcrumbs.php#L52, as `$tree` is got from cache that is old, `$tree[$code]` does not exist and the loop never ends.

The reason is: the cache is written with a key `-902817884static-page-menu-en`, but after page changes and saved the plugin tries to clear cache using key `-902817884static-page-menu-ru`. The language code is different and it leads to incorrent file paths, so cache is never cleared.

I have got this details debugging `$key` variable in `\Illuminate\Cache\FileStore::put` and `\Illuminate\Cache\FileStore::forget`

**Comment by daftspunk** [comment]
Can you submit a Pull Request to fix it?

**Comment by side-by-side** [comment]
Not for now, i haven't yet figured out where this language code coming from.

**Comment by side-by-side** [comment]
I have made some research.
What's going on:
The static pages cache is written on page request in frontend. The key for that cache entry is using the language that set in the config ```app.locale``` that is ```en``` by default. So the cache key looks like ```-902817884static-page-menu-en```.

When we have changed static page in backend, cache is suppose to be cleared. The cache key is generating using
```php
Lang::getLocale()
```
that is match the **current user backend session language**, which may be not default one from the ```app.locale```. In my case it is ```ru``` and the cache key looks like ```-902817884static-page-menu-ru```.

The potential solution is to clear cache using the default locale from ```app.locale```. But in backend the locale is changed and the default config value is overrided:
1. https://github.com/octobercms/october/blob/develop/modules/backend/classes/Controller.php#L221
2. https://github.com/octobercms/october/blob/develop/modules/backend/models/Preferences.php#L61
3. https://github.com/laravel/framework/blob/5.2/src/Illuminate/Foundation/Application.php#L1037

To implement this solution i need the original locale value from ```app.locale``` and i don't know how to get it instead of the one from the current session.

Also i think there may be related problems with multilingual sites. I am working with Laravel and October just a few days, switching from the Symfony for the project, and trying my best.
